### PR TITLE
865 individual custom field edm patch

### PR DESCRIPTION
**Pull request by drewblount**
Adds support for patching individual custom fields. Very simple so IMO merge-able after one reviewer checks off.

One-line fix as identified by Emily and Jon in DEX-865, along with a test replicating the reported bug. The test simply creates an individual, creates a custom field (these fields are registered in the EDM db), and then sets that custom field for the individual.

Before the one-line fix, the test broke. After the one-line fix, the test passes.

### CODE REVIEW

**Comment by drewblount** [comment]
If there are issues with automated testing, we need to wait until the latest EDM PR is merged into next-gen and rerun.

**Review by hwindsor — APPROVED** [review]
LGTM

**Pull request merged**
